os/kernel/binary_manager/compression/Make.defs

```diff
@@ -0,0 +1,28 @@
+###########################################################################
```

Comment:
The Kconfigs for compression are defined in os/Kconfig , the source code is present in binary_manager and it is being used by both binary manager and binary loader. So, how about making the compression logic as a separate module under os/compression?

Reply:
The reason for putting compression code under binary manager is that without binary manager, we do not need to compile compression related logic. Our compression related logic only makes sense to be compiled if binary manager is enabled for handling binaries. Hence, the dependency.